src/makeCalculator.js

```diff
@@ -4,7 +4,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: function (valueResult) {
+      return function (value) {
+        return valueResult + value;
+      };
+    },
+
+    subtract: function (valueResult) {
+      return function (value) {
+        return valueResult - value;
+      };
+    },
+
+    multiply: function (valueResult) {
+      return function (value) {
+        return valueResult * value;
+      };
+    },
+
+    divide: function (valueResult) {
+      return function (value) {
+        if (value > 0) {
+          return valueResult / value;
+        } else {
+          return 'Error';
+        }
```

Comment:
The check for `value > 0` is incorrect for division. You should check if `value` is not equal to 0 to prevent division by zero errors. Also, consider throwing an error or handling the error case differently than returning a string.